dvc/command/version.py

```diff
@@ -129,7 +128,12 @@ def get_supported_remotes():
             if not tree_cls.get_missing_deps():
                 supported_remotes.append(tree_cls.scheme)
 
-        return ", ".join(supported_remotes)
+        if len(supported_remotes) == len(TREES):
+            return "All remotes"
+        elif len(supported_remotes) == 1:
+            return supported_remotes
+        else:
+            return ", ".join(supported_remotes)
```

Comment:
@sahilbhosale63, the plan was to make `Supports` use both remotes and cache types. But, as we didnot go through that, I think it's okay to not go "All remotes" route. But, it's your call.